### PR TITLE
Some minor improvements

### PR DESCRIPTION
**Pull request by dandjo**
The major change is the optimization of not bloating the MQTT json message when using OneATTR. The readout is faster and the network doesn't transfer redundant information.

The implementation of my COP calculation in openHAB (ECMA script) is untested due to lacking temperatures for heating in the summer. Note that the values read are my configured Items in openHAB. It needs editing to make it work.

Of course you can cherrypick features if you like my stuff.

### CODE REVIEW

**Comment by raomin** [comment]
Hi @dandjo,

Thanks for this, I'm a little busy now but I'll have a look this week!

File: src/main.cpp [hunk 1]
```diff
@@ -255,10 +254,10 @@ void loop()
     {
       converter.readRegistryValues(buff); //process all values from the register
       updateValues(registryIDs[i]);       //send them in mqtt
-      waitLoop(500);//wait .5sec between registries
+      //waitLoop(500);//wait .5sec between registries
```
Comment (dandjo) [inline src/main.cpp:257]: I'm not sure whether and why a waitLoop here is necessary.

File: src/main.cpp [hunk 2]
```diff
@@ -255,10 +254,10 @@ void loop()
...
     }
   }
   sendValues();//Send the full json message
-  mqttSerial.printf("Done. Waiting %d sec...\n", FREQUENCY / 1000);
+  mqttSerial.printf("Done. Waiting %d sec...", FREQUENCY / 1000);
```
Comment (dandjo) [inline src/main.cpp:261]: The newline lead to serious problems with some MQTT listeners. So removing for compatibility.

File: src/main.cpp (hunk 1 above)
Comment (raomin) [reply src/main.cpp:257]: That was maybe an unnecessary precaution... OK to remove it.

**Review by raomin — APPROVED** [review]
Ok for me, thanks.

**Comment by dandjo** [comment]
Nice!

**Pull request merged**